Sources/CertLogic/CertLogic.swift

```diff
@@ -15,6 +15,7 @@ public enum ValidationType {
   case issuer
   case destination
   case traveller
+  case custom
```

Comment:
Is there really no better name than "custom"? Or add a comment what you mean with it.